setup.py

```diff
@@ -16,6 +16,7 @@
 
 required_packages = [
     # Common packages
+    "py-spy==0.3.11",
```

Comment:
No because it's useful on both platforms.

However, even this kind of packages should not be included in the requirements because it's not used in the code and only for development (so it could just be added in the local development venv), I added it to be sure it will be avalaible in Yocto build if we need to analyze performance issues.